c8/fabric.py

```diff
@@ -156,22 +157,23 @@ def on_change(self, collection, callback, timeout=60):
 
         subscription_name = "%s-%s-subscription-%s" % (
             self.tenant_name, self.fabric_name, str(random.randint(1, 1000)))
-        
+
         url = self.url.split("//")[1].split(":")[0]
 
         topic = "wss://{}/_ws/ws/v2/consumer/persistent/{}/{}/{}/{}".format(
-            url,self.tenant_name,namespace,
-            collection,subscription_name)
+            url, self.tenant_name, namespace,
+            collection, subscription_name)
+
+        ws = websocket.create_connection(
+            topic, header=self.header, timeout=timeout)
 
-        ws = websocket.create_connection(topic, header=self.header, timeout=timeout)
-        
         try:
             print("pyC8 Realtime: Begin monitoring realtime updates for " +
                   topic)
             while True:
                 msg = json.loads(ws.recv())
-                data =  base64.b64decode(msg['payload'])
-                ws.send(json.dumps({'messageId' : msg['messageId']}))
+                data = base64.b64decode(msg['payload'])
```

Comment:
JSON schema should be validated before parsing it. Driver should address possible changes in the backend. Please validate the existence of 'payload' key in the dictionary before processing anything with it.